cs/course.md

```diff
@@ -71,7 +71,7 @@ Semestrální práce
 Instalace programů
 ------------------
 
--   viz [Programy](apps.md)
+-   viz Programy (TODO dát link)
```

Comment:
I'm not very happy with this, but let's just keep the linter happy for now, until we have it.

Reply:
I understand, but I believe that the leverage of having all links working just might overweight this...

You can always overcome this by creating empty target file, having only heading...